test/unit/clear-refinement.ts

```diff
@@ -8,21 +9,48 @@ suite('ClearRefinement', ({ expect, spy, itShouldProvideAlias }) => {
 
   itShouldProvideAlias(ClearRefinement, 'clearRefinement');
 
-  describe('constructor()', () => {
-    describe('state', () => {
-      describe('onClick()', () => {
-        it('should call actions.deselectRefinement()', () => {
-          const field = 'brand';
-          const index = 13;
-          const deselectRefinement = spy();
-          clearRefinement.props = { field, index };
-          clearRefinement.actions = <any>{ deselectRefinement };
-
-          clearRefinement.state.onClick();
-
-          expect(deselectRefinement).to.be.calledWith(field, index);
-        });
-      });
+  describe('init()', () => {
+    let field, index;
```

Comment:
Our linter doesn't allow multiple variable declarations. Please put these on a separate line and use `let` for each variable

Reply:
also because you're actually using these as consts it should be
`const field = 'brand';`
`const index = 13;`
remove them from the beforeeach